process/best_practices_badge.md

```diff
@@ -0,0 +1,57 @@
+---
+parent: Processes
+has_children: true
+---
+
+# OpenSSF Best Practices Badge
+
+* TOC
+{:toc}
+
+
+The [Open Source Security Foundation (OpenSSF) Best Practices badge](https://bestpractices.coreinfrastructure.org/en) allows Free/Libre and Open Source Software (FLOSS) projects to show that they follow best practices in managing their project. Consumers of the badge can quickly assess which FLOSS projects are following best practices and, as a result, are more likely to produce higher-quality secure software. Academy Software Foundation projects, as a part of the [project lifecycle established by the TAC](lifecycle), achieve this badge to showcase them being exemplars in open source for the visual effects and motion picture industries. 
+
+This document outlines specific questions Academy Software Foundation projects have raised in regard to fulfilling the badge requirements. If you have questions not answered, please provide a PR that adds your question below and will be merged with an answer.
+
+## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+
+Having a primary developer take the three courses outlined below ( all free to audit ) would be sufficient:
+
+- [Secure Software Development: Requirements, Design, and Reuse (LFD104x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-requirements-design-and-reuse-lfd104/) 
+- [Secure Software Development: Implementation (LFD105x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-implementation-lfd105/) 
+- [Secure Software Development: Verification and More Specialized Topics (LFD106x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-verification-and-more-specialized-topics-lfd106/) 
+
+## Is there a list or document which describes the static analysis tools that ASWF supports or recommends?
+
+SonarCloud is the tool that multiple projects have implemented, and there is already an organization-level access token that projects can use with it. You can see a specific example in the OpenEXR analysis_workflow.yml, which shows how to build, run the test suite, post-process, and upload to SonarCloud using the SONAR_TOKEN from the organization level. You can see the results for the various ASWF projects with SonarCloud here:
+
+[https://sonarcloud.io/organizations/academysoftwarefoundation/projects](https://www.google.com/url?q=https://sonarcloud.io/organizations/academysoftwarefoundation/projects&sa=D&source=docs&ust=1675375752427760&usg=AOvVaw3aJ1DsMv9yF2Xf-Uhv8EKo)
```

Comment:
This target of this link got wrapped by the google search, it should just be:

https://sonarcloud.io/organizations/academysoftwarefoundation/projects